dashboard/app/views/levels/editors/_all.html.haml

```diff
@@ -10,7 +10,7 @@
   - script_data = { editorall: { video_infos: video_infos }.to_json }
   %script{src: minifiable_asset_path('js/levels/editors/_all.js'), data: script_data}
 
-- unless @level.is_a? DSLDefined
+- unless @level.is_a?(DSLDefined) || @level.is_a?(Unplugged)
```

Comment:
I noticed this change (extending `@level.is_a?(DSLDefined)` to `@level.is_a?(DSLDefined) || @level.is_a?(Unplugged)` is being added in several places, suggesting that `DSLDefined` and `Unplugged` level types are related in a way that isn't reflected in our class hierarchy.

Would it simplify the logic to insert another layer into the Level hierarchy that represents whatever concepts are shared between `Unplugged` and `DSLDefined` types? (I don't know what it would be called- `NonPuzzleLevel`?)

It's not currently clear to me why (for example) only `DSLDefined` and `Unplugged` level types (and not any other level types) don't have the ability to set `display_name` and `notes` in the editor form.